angular-autocomplete.css

```diff
@@ -5,7 +5,6 @@
     display: inline-block;
     background: #fff;
     min-width: 100%;
-    max-width: 100%;
```

Comment:
This is to make the width to auto expand if the item width exceeds that of the parent.